flask_app/templates/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MPTCP Connection Status</title>
+    <link rel="stylesheet" type="text/css" href="/static/styles.css">
+</head>
+<body>
+    <a id="logo" href="https://mptcp.dev"><img src="/static/MPTCP_logo.svg"></a>
```

Comment:
can you not use it from `https://www.mptcp.dev/assets/mptcp_logo.svg` instead? (and remove it from the static dir)

Reply:
I mean: you don't need to include the logo in this repo, simply use the one from `mptcp.dev`, no?

Reply:
I did it for 'redundancy' but if you believe it is fine. It is fine by me.

Reply:
up to you, fine either way.